tests/unit/algorithms/maml_test.py

```diff
@@ -188,6 +188,35 @@ def forward(self, x):
         loss = sum(p.norm(p=2) for p in clone.parameters())
         loss.backward()
 
+    def test_memory_consumption(self):
+        def get_memory():
+            return torch.cuda.memory_allocated(0)
+
+        BSZ = 1024
+        INPUT_SIZE = 128
+        N_STEPS = 5
+        N_EVAL = 2
+
+        device = torch.device('cuda:0')
```

Comment:
GitHub actions don't have cuda. Can we shield this test with: `if torch.cuda.is_available():` so it doesn't run if the machine doesn't have cuda?

Reply:
fixedd